recipe/meta.yaml

```diff
@@ -11,13 +14,15 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
Instead of increasing the build number, please update the version. This way is easier to control which version will be used to build ffmpeg.